windows/sound.cpp

```diff
@@ -109,8 +109,15 @@ strCurDevName = cDriverNames[iDriverIdx];
 void CSound::UnloadCurrentDriver()
 {
     // clean up ASIO stuff
-    ASIOStop();
-    ASIODisposeBuffers();
+    if ( bRun )
+    {
+        this->Stop();

```

Comment:
This can just be `Stop();` can't it? (no need for the redundant `this->`)

Reply:
Oh right.